src/routes/embedded/auth-request/connect/connect.view.tsx

```diff
@@ -52,7 +52,7 @@ export function EmbeddedConnectAuthRequestView() {
   return (
     <>
       <AuthRequestCard
-        onCancel={rejectRequest}
+        onCancel={() => rejectRequest()}
```

Comment:
Otherwise, we are passing the click event to the accept/reject handler, which actually have arguments (data or error, respectively).